etc/run.sh

```diff
@@ -2,4 +2,9 @@
 
 alembic upgrade head
 python load_csv.py --path ${DIALOG_DATA_PATH}
-uvicorn main:app --host 0.0.0.0 --port ${PORT}
+
+if "${DEBUG}"; then
+    uvicorn main:app --host 0.0.0.0 --port ${PORT} --reload
+else
+    uvicorn main:app --host 0.0.0.0 --port ${PORT}
+fi
```

Comment:
I think that inject `UVICORN_RELOAD=true` as environment variable on container solves the problem